localMatchApp/signup.cpp

```diff
@@ -0,0 +1,39 @@
+#include "signup.h"
+#include "ui_signup.h"
+#include "QDesktopWidget"
+#include <QMessageBox>
+
+SignUp::SignUp(QWidget *parent) :
+    QDialog(parent),
+    ui(new Ui::SignUp)
+{
+    ui->setupUi(this);
+    ui->lineEdit_NewUser->setPlaceholderText("Username");
+    ui->lineEdit_NewPassword->setPlaceholderText("Password");
+    ui->lineEdit_NewPassCheck->setPlaceholderText("Repeat Password");
+    QPixmap pix ("/home/marcelino/Desktop/LocalMatch/images/logo.png");
+    ui->picture->setPixmap(pix);
+    this->setFixedWidth(500);
+    this->setFixedHeight(450);
+    this->setGeometry(
+        QStyle::alignedRect(
+            Qt::LeftToRight,
+            Qt::AlignCenter,
+            this->size(),
+            qApp->desktop()->availableGeometry()
+        )
+    );
+}
+
+SignUp::~SignUp()
+{
+    delete ui;
+}
+
+void SignUp::on_NewUserButton_clicked()
+{
+    QMessageBox* msgbox = new QMessageBox(this);
```

Comment:
Memory leak. Please, contemplate the option of using smart pointers here or any other RAII approach.